build.gradle.kts

```diff
@@ -32,9 +32,9 @@ plugins {
     `maven-publish`
     signing
     checkstyle
-    id("de.fuerstenau.buildconfig") version "1.1.8"
-    id("com.github.spotbugs") version "4.6.0"
-    id("com.diffplug.spotless") version "5.10.0"
+    id("com.github.gmazzo.buildconfig") version "3.0.3"
```

Comment:
Had to migrate to an alternative plugin that doesn't use removed dependency types (compile)